cmd/layout_image_handler_test.go

```diff
@@ -0,0 +1,90 @@
+package cmd_test
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/sclevine/spec"
+	"github.com/sclevine/spec/report"
+
+	"github.com/buildpacks/lifecycle"
+	"github.com/buildpacks/lifecycle/cmd"
+	h "github.com/buildpacks/lifecycle/testhelpers"
+)
+
+func TestLayoutImageHandler(t *testing.T) {
+	spec.Run(t, "VerifyAPIs", testLayoutImageHandler, spec.Sequential(), spec.Report(report.Terminal{}))
+}
+
+func testLayoutImageHandler(t *testing.T, when spec.G, it spec.S) {
+	var (
+		imageHandler lifecycle.ImageHandler
+		registry     string
+		layoutDir    string
+		imageRef     string
+	)
+
+	when("default docker registry", func() {
```

Comment:
This is interesting... so it sounds like we DO want to allow users to "mix and match" inputs (provide docker image as input, get OCI layout as output). In this case we may want to revisit the `oci:/path/to/my/image` interface...

Reply:
What about when the image handler is provided an OCI layout image?

Reply:
Note to future selves: when the input image is a daemon image, we'll want to use the launch cache or else it will be very slow.

Reply:
One last thought - let's move this and the other image "handler" from `main` into the `image` package... it will then hold methods and interfaces that are wrappers around `imgutil.Image`, which makes sense organizationally.